metadata/src/main/java/edu/unc/lib/dl/util/DepositStatusFactory.java

```diff
@@ -59,9 +58,9 @@ public DepositStatusFactory() {}
 	public Set<Map<String, String>> getAll() {
 		Set<Map<String, String>> result = new HashSet<>();
 		try (Jedis jedis = getJedisPool().getResource()) {
-			Set<String> deposits = jedis.smembers(DEPOSIT_SET);
-			if (deposits != null) {
-				for(String uuid : deposits) {
+			Set<String> keys = jedis.hkeys(DEPOSIT_STATUS_PREFIX +"*");
+			if (keys != null) {
+				for(String uuid : keys) {
```

Comment:
Might as well add the space after for while I'm being pedantic, although I know it was already this way.